demo.js

```diff
@@ -0,0 +1,8 @@
+const something = "im a bad thing"
+const breakingChange = something.notValid
+
+const newThing = "a thing ive never seen before"
+
+const oldWay = "old way of doing something"
```

Comment:
```suggestion
const newWay = "new way to do thing"
```
Suggestion: Instead of old way, perhaps new way might be better here?